themes/default/views/editor/objects/summary_html_ajax.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/* ----------------------------------------------------------------------
```

Comment:
@kehh is this what we want this content to look like for new templates we submit upstream?